source/_integrations/homematic.markdown

```diff
@@ -485,16 +485,16 @@ template:
   - binary_sensor:
       - name: "Homematic is sending updates"
         state: >-
-          {{ now() - as_timestamp(state_attr('sensor.office_voltage', 'last_changed'), 601) < 600 }}
+          {{ (now() - states.sensor['office_voltage'].last_changed).seconds < 600 }}
```

Comment:
```suggestion
          {{ (now() - states.sensor.office_voltage.last_changed).seconds < 600 }}
```

Reply:
This doesn't work in my case, as the given sensor has a generated entitiy-id is starting with numbers. But for the documentation this is definetly the cleaner version.

Reply:
Entity IDs cannot start with a number.

Validation rule: `^(?!.+__)(?!_)[\da-z_]+(?<!_)\.(?!_)[\da-z_]+(?<!_)$`

Reply:
I'm using an homematic ccu3. The generated id's look like `00095be98bXXXX_illumination`.

If I use this in a template
`{{ (now() - states.sensor.00095be98bXXXX_illumination.last_updated).seconds < 3600 }}`

I run into
`Invalid config for [template]: invalid template (TemplateSyntaxError: expected token ')', got 'integer') for dictionary value @ data['binary_sensor'][0]['state']. Got '{{ (now() - states.sensor.00095be98bXXXX_illumination.last_updated).seconds < 3600 }}'`